### ISSUE TITLE
Bug when not logged in clicking refresh

### ISSUE THREAD

**Issue by FinalDoom** [FinalDoom/FFXI-FishingForm#29]
**[Original report](https://bitbucket.org/FinalDoom/ffxi-fishingform/issue/29) by FinalDoom (Bitbucket: [FinalDoom](https://bitbucket.org/{d7bd602f-3443-4a67-a621-ea4c07d584c2}), GitHub: [FinalDoom](https://github.com/FinalDoom)).**

----------------------------------------

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at Fishing.FishingForm.IsRodBaitEquipped()
   at Fishing.FishingForm.btnRefreshLists_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by FinalDoom** [comment]
**Original comment by FinalDoom (Bitbucket: [FinalDoom](https://bitbucket.org/{d7bd602f-3443-4a67-a621-ea4c07d584c2}), GitHub: [FinalDoom](https://github.com/FinalDoom)).**

----------------------------------------

Also when clicking send. Needs some null checking prolly.
```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at Fishing.FishingForm.btnChatSend_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by FinalDoom** [comment]
**Original comment by FinalDoom (Bitbucket: [FinalDoom](https://bitbucket.org/{d7bd602f-3443-4a67-a621-ea4c07d584c2}), GitHub: [FinalDoom](https://github.com/FinalDoom)).**

----------------------------------------

[fix] fixes #29
